src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IIsPatternExpression.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
-    public partial class IOperationTests : SemanticModelTestBase
+    public partial class IOperationTests_Patterns : SemanticModelTestBase
```

Comment:
Consider syncing the file name.

Reply:
Also, breaking up the test classes has been on my backlog for a while. If you're going to do this here, consider renaming to a better name, something like `PatternsTests`. We're already in an `IOperation` folder in an `IOperation` tests project, so it should be plenty disambiguated.

---
In reply to: [244201848](https://github.com/dotnet/roslyn/pull/32018#discussion_r244201848) [](ancestors = 244201848)